### PR TITLE
EnglishAnalyzer for strings.

### PR DESCRIPTION
**Pull request by elcct**
This can be used for strings that contain English text to be searchable.

### CODE REVIEW

**Comment by reupen** [comment]
Could probably stick it on company description as well.

File: datahub/search/elasticsearch.py [hunk 1]
```diff
@@ -31,6 +30,35 @@
...
+    'english_stop',
+    type='stop',
+    stopwords='_english_'
+)
+
+english_analyzer = analysis.CustomAnalyzer(
+    'english_analyzer',
+    tokenizer='standard',
+    filter=[
+        english_possessive_stemmer,
```
Comment (reupen) [inline datahub/search/elasticsearch.py]: Do we need the `english_possessive` stemmer if we're using the plain `english` stemmer?

Comment (elcct) [reply datahub/search/elasticsearch.py]: The doc says: `The possessive stemmer removes 's from any words before passing them on to the english_stop, english_keywords, and english_stemmer.`.

**Comment by elcct** [comment]
I added it to description fields across models as well.

**Pull request merged**